### PR TITLE
Issue#21 - Tron circulating supply

### PR DESCRIPTION
**Pull request by PaddyMc**

### CODE REVIEW

File: blockchain-scrapers/docker-compose.yml [hunk 1]
```diff
@@ -96,6 +96,24 @@ services:
       restart_policy:
         delay: 2s
         window: 20s
+  tron:
+    image:
+      ${DOCKER_HUB_LOGIN}/tron
```
Comment (frranck) [inline blockchain-scrapers/docker-compose.yml]: you need to remove the ${DOCKER_HUB_LOGIN}, it gets me a "No such image", I don't have this image in my repository. what is the image you want to get ?

Comment (PaddyMc) [reply blockchain-scrapers/docker-compose.yml]: Hey @frranck, 

You can target "patches12345/tron", the image is located here: https://hub.docker.com/r/patches12345/tron/

It's the image built from this repo: https://github.com/PaddyMc/go-tron/blob/master/Dockerfile

I can add my Docker Hub username to the docker-compose if you would like, but my advice would be to build the docker image linked above and upload it to your Docker Hub and use that for production. 

Let me know how you would like to proceed.

Comment (frranck) [reply blockchain-scrapers/docker-compose.yml]: Could you please include your docker file for tron into the project ?
then use:
image: ${DOCKER_HUB_LOGIN}/blockchain-scrapers_tron

Comment (PaddyMc) [reply blockchain-scrapers/docker-compose.yml]: Ok, it's done. I've put the Tron node Dockerfile in the cmd/trx folder. And updated the tron image location to ${DOCKER_HUB_LOGIN}/blockchain-scrapers_tron

Just to be clear, there are two Dockerfiles now.

- one for the tron node (tron) => cmd/trx/Dockerfile used in the docker-compose.yml (line 101)
- one for the blockchain scraper (trx) => Dockerfile-trx used in the docker.compose.yml (line 218)

So in the the docker-compose there are two services, tron & trx

Let me know if there's anything else I can do to help, more than happy to.

**Comment by frranck** [comment]
Hello Paddy,

Can't build your trx image:

> Building trx
> Step 1/10 : FROM golang:latest as build
>  ---> d0e7a411e3da
> Step 2/10 : RUN go get github.com/PaddyMc/go-tron/calculations
>  ---> Running in 436be6259622
> Removing intermediate container 436be6259622
>  ---> 2bf80db2dfb8
> Step 3/10 : RUN go get github.com/diadata-org/api-golang/dia
>  ---> Running in e68020b1138c
> Removing intermediate container e68020b1138c
>  ---> 840f1fedb9fc
> Step 4/10 : WORKDIR $GOPATH/src/
>  ---> Running in 9d3643f1a9b2
> Removing intermediate container 9d3643f1a9b2
>  ---> ceb2d848b755
> Step 5/10 : COPY . .
>  ---> 61a838a99ed6
> Step 6/10 : WORKDIR $GOPATH/src/cmd/trx
>  ---> Running in 28776e517884
> Removing intermediate container 28776e517884
>  ---> 65a8a3e89b22
> Step 7/10 : RUN go install
>  ---> Running in 26a6891054b3
> can't load package: package cmd/trx: no Go files in /go/src/cmd/trx
> ERROR: Service 'trx' failed to build: The command '/bin/sh -c go install' returned a non-zero code: 1
> 

also could you copy the tron Dockerfile with the other dockerfiles ? rename it and refer to it in the docker-compose

> 
>   tron:
>     build:
>       context: ../../../..
>       dockerfile: github.com/diadata-org/api-golang/blockchain-scrapers/Dockerfile-tron
>     image:
>       ${DOCKER_HUB_LOGIN}/blockchain-scrapers_tron
> 

thanks

**Comment by PaddyMc** [comment]
Done and done, third times a charm! 

These issues have been fixed, it should run now. 

Let me know if you run into anymore issues and I'll be happy to fix them.

Thanks!

**Pull request merged**